dspace-api/src/main/java/org/dspace/curate/CurationCli.java

```diff
@@ -104,6 +109,19 @@ public static void main(String[] args) throws Exception {
             ePersonName = line.getOptionValue('e');
         }
 
+        if (line.hasOption('p')) { // parameter
+            for (String parameter : line.getOptionValues('p')) {
+                String[] parts = parameter.split("=", 2);
+                String name = parts[0];
+                String value;
+                if (parts.length > 1) {
+                    value = parts[1];
+                } else {
+                    value = "true";
+                }
+                parameters.put(name, value);
```

Comment:
Wouldn't it make sense to trim both `name` and `value` here in case the data is entered as e.g. `'NAME = VALUE'`?

Reply:
That can only happen if the entire expression is quoted:  ``-p 'foo = bar'``, which would be rather silly, but this is still a sensible suggestion.  (Tested:  unquoted spaces are argument delimiters, so ``-p foo = bar`` results in an option 'p' with value 'foo' and two non-option arguments '=' and 'bar'.)

My test classes tend to have commented-out tests because my IDE creates them all but I have developed the bad habit of only implementing the ones I need at the time.  For now I will confirm my habit by removing all of the unused methods.